tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -4,6 +4,7 @@ namespace FakeItEasy.Specs
     using System.Diagnostics.CodeAnalysis;
     using System.Threading.Tasks;
     using FakeItEasy.Configuration;
+    using FakeItEasy.Creation;

```

Comment:
Hey! FakeItEasy.Creation! I wasn't confused in my comment on the capturing issue. Just prescient.